hypothesis-python/tests/cover/test_float_utils.py

```diff
@@ -35,3 +41,43 @@ def test_next_float_equal(func, val):
         assert math.isnan(func(val))
     else:
         assert func(val) == val
+
+
+@pytest.mark.parametrize(
+    "minfloat,maxfloat",
+    [
+        (0.1, 5.0),
+        (0.0, math.inf),
```

Comment:
This seems like a great case for a property-based test!

(I mean, the test is already property-based, but putting these in `@example()` cases and using `@given()` to generate more)

Oh, and let's also assert that `if minfloat <= value <= maxfloat or (allow_zero and value == 0): assert clamper(value) is value`

Reply:
:) I wasn't sure how shy to be about using float generation to test float generation! But took a stab at it, and it feels good, and the `example()`s should catch fundamental issues.